src/main/drivers/accgyro_mpu.c

```diff
@@ -234,8 +234,7 @@ static bool detectSPISensorsAndUpdateDetectionResult(gyroDev_t *gyro, const sens
 #else
     UNUSED(bus);
 #endif
-    if (mpu6000SpiDetect(&gyro->bus)) {
-        gyro->mpuDetectionResult.sensor = MPU_60x0_SPI;
+    if ((gyro->mpuDetectionResult.sensor = mpu6000SpiDetect(&gyro->bus))) {
```

Comment:
`mpu6500SpiDetect` case needs it, because `mpu6500SpiAccDetect` and `mpu6500SpiGyroDetect` checks what `mpu6500SpiDetect` found.
Of course, this is based on the assumption that `mpuDetectionResult.sensor` is further used somewhere else; if not, we can make `mpu6500SpiDetect` to return `bool`.